src/com/github/kornilova203/matlab/psi/MatlabStubbedFunctionDeclaration.kt

```diff
@@ -54,7 +58,19 @@ abstract class MatlabStubbedFunctionDeclaration : StubBasedPsiElementBase<Matlab
     }
 
     override fun getType(): MatlabType {
-        val cl = this.isConstructor()
+        val cl = this.isConstructor() ?: this.isClassReturn()
         return if (cl != null) MatlabTypeClass(cl) else MatlabTypeFunction(this)
```

Comment:
It's strange that function declaration has class type.
It's better to return `MatlabTypeFunction` that returns class type in `getReturnType` method